### PR TITLE
Fix rendering in ElementHost with SkiaSharp

### LINKED ISSUE #1800: PlotView in ElementHost does not work.
### Steps to reproduce

I included a repo as a Zip with this issue, to make it easier to reproduce the issue. Just hit start and you should be able to see that the PlotView is not rendered inside a UserControl, which is contained in a ElementHost-Control. If i execute the same code without the ElementHost inside a WPF-Window-Control, the Plot is appearing just fine.

OxyPlot version: 2.1.0
Platform: WPF
.NET version: .NET Framework 4.7.2

### Expected behaviour

I am expecting, that the PlotView is rendered and my plot is shown.
### Actual behaviour

The PlotView stays blank.

[OxyPlotElementHostTest.zip](https://github.com/oxyplot/oxyplot/files/7323145/OxyPlotElementHostTest.zip)

### PR DESCRIPTION
**Pull request by AELIENUS**
Fixes #1800  .

### Checklist

- [ ] I have included examples or tests
- [x] I have updated the change log
- [x] I am listed in the CONTRIBUTORS file
- [ ] I have cleaned up the commit history (use rebase and squash)

### Changes proposed in this pull request:

- Pulled over changes from #1797 in GetAncestorFromVisualTree in PlotView.cs to OxySKElement.cs (related issue #1796)
- Added the AncestorVisual for ElementHosts to IsInVisualTree() in PlotViewBase.cs

@oxyplot/admins

### CODE REVIEW

**Comment by AELIENUS** [comment]
I could not get it to run with the "normal" WPF-Package. I needed SkiaSharp to get it to run in the ElementHost

**Comment by VisualMelon** [comment]
Thanks for putting this together. I'll try to take a proper look at it in the next few days.

**Review by VisualMelon — APPROVED** [review]

**Comment by VisualMelon** [comment]
Overlap with #1799

**Comment by AELIENUS** [comment]
#1799 looks good to me in combination with this change. I don't think, that anything is missing, but I also don't know all UseCases. I guess time will tell.

**Comment by VisualMelon** [comment]
@AELIENUS thanks looking at #1799 

I've gone through and checked most of the open PRs; I'll give the other maintainers a bit of time to look, but assuming there are no objections I will (hopefully) start merging them in a few days.

**Comment by AELIENUS** [comment]
I have overlooked this again after #1799 was already merged. There were two files with conflicts: the changelog (that was rather easy to fix and i really dont understand why git flagged this as conflict in the first place) and in  OxySKElement (there were inconsistencies with the documentation  of   GetAncestorVisualFromVisualTree there that i fixed in my branch).

File: Source/OxyPlot.Wpf.Shared/PlotViewBase.cs
Comment (VisualMelon) [inline Source/OxyPlot.Wpf.Shared/PlotViewBase.cs:462]: What scenario does this cover? (I've hardly used WPF in years, and not familiar with adorners)

Comment (AELIENUS) [reply Source/OxyPlot.Wpf.Shared/PlotViewBase.cs:462]: This is the parent of the ElementHost. It is used to wrap WPF in Windows Forms

**Comment by AELIENUS** [comment]
@VisualMelon Can we merge this PR now or do you need more time or assistance?

**Comment by VisualMelon** [comment]
Was giving other maintainers a chance to look at it, merged now.

**Pull request merged**
